docs/settingUp.md

```diff
@@ -111,6 +113,24 @@ Eclipse IDE is our preferred development environment. Support requests related t
    * Set the `root directory` to the location where the repo is cloned.
    * Click `Finish`.
 
+### IntelliJ IDEA <a name="intellij"></a>
+1. Download and install [IntelliJ IDEA Ultimate Edition](https://www.jetbrains.com/idea/).
+   * You will need an Ultimate Edition license in order to work with Google App Engine in IntelliJ IDEA.
+   * Sign up for the free [JetBrains student license](https://www.jetbrains.com/student/) if you are a registered student.
+1. Configure IntelliJ
+   * Click `Configure → Project Defaults → Project Structure`.
+   * Under `Project SDK`, click `New → JDK`. Locate the `Java` folder where you have installed the `JDK 1.7`.
+   * Select `jdk1.7.0_80` and click `OK`.
```

Comment:
How about moving this step to after importing the project? Changing the Project Defaults for a single project seems a little odd.